openhtf/__init__.py

```diff
@@ -334,19 +364,19 @@ class PhaseInfo(mutablerecords.Record(
 
   @classmethod
   def WrapOrCopy(cls, func):
```

Comment:
now that `PhaseDescriptor.WrapOrCopy` is so long, and it gets used by test authors (to call `WithArgs` on a phase that doesn't use plugs, for instance), could we get a better-named function that calls out to this? Maybe `AsPhase`?